content/source/docs/extend/guides/v2-upgrade-guide.html.md

```diff
@@ -509,6 +525,36 @@ this will involve changing the returned type of the `Provider()` function, and
 removing some type assertions; all necessary changes should be raised at
 compile time.
 
+## Removal of the `helper/schema.Provider.StopContext` method
+
+The `helper/schema.Provider.StopContext` method has been removed as its
+implementation has been reconfigured. Use `helper/schema.StopContext`, passed a
```

Comment:
Sentence is a bit funky, but it's only safe to use `schema.StopContext` during `ConfigureContextFunc` (or whatever it's actually called)